proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -44,157 +44,127 @@ direction of the policies set by the Board.
+
+### Regular members
+
+Regular members have the roles and responsibilities
+outlined below. Regular member is implied when memership type is not
+specified. Anyone who has been a member of one of the projects under the 
+Foundation for at least three months may request to
+become a regular member by opening a PR to add themselves to the list of
+regular members. Regular members remain for as long as they are active
+within the work of the CPC. Regular members who have not been active
+in GitHub, participated in meetings, or other work of the CPC for 3 months may be
+removed from the list of Regular members. In addition, a Regular CPC member
+can be removed by voluntary resignation, or by a standard CPC motion.
+
+### Voting members
+
+Voting members are selected as follows:
+
+* Each top level project may nominate two members through a process
+of their choosing. Once nominated the member must be ratified by the
+board before becoming a Voting member.
+
+* two Voting members may be nominated by the non top-level projects based
+on a process set by the CPC.
+
+* two Voting members may be nominated by the Regular members. Once
+nominated these members must be ratified by the board before
+becoming a Voting member.
+
+Voting members are expected to make a time commitment which
+allows them to be responsive to CPC business, participate regularly
+in meetings and to participate in all voting matters (either by
+voting or specifically abstaining).  They are
+also expected to help to enable work of the regular CPC
+members by providing leadership, help with interactions with the
+board and Foundation staff and to generally help keep things moving.
+
+Voting members serve for a term of 1 year and must be re-nominated
+and ratified by the board each year. In addition, a Voting CPC member
+can be removed by voluntary resignation, by a standard CPC motion,
+or in accordance to the participation rules described for Regular
+members.
+ 
 Changes to CPC membership should be posted in the agenda, and may be suggested
 as any other agenda item.
 
-No more than one-fourth of the CPC members may be affiliated with the
-same employer. If removal or resignation of a CPC member, or a change of
-employment by a CPC member, creates a situation where more than
-one-fourth of the CPC membership shares an employer, then the situation
-must be immediately remedied by the resignation or removal of one or more
-PC
+No more than one-fourth of the Voting CPC members may be affiliated with the
+same employer. Removal or resignation of a Voting CPC member, or a change of
+employment by a Voting CPC member could cause more than
+one-fourth of the Voting CPC membership to be affiliated with the same emplyer.
+Such a situation must be immediately remedied by the resignation or
+removal of one or more voting CPC members.
 
 The CPC may, at its discretion, invite any number of non-voting observers to
 participate in the public portion of CPC discussions and meetings.
```

Comment:
All contributors of all projects should be invited to participate in all public sessions. It should be part of the charter for the CPC to encourage active participation

Reply:
Updated to indicate that through https://github.com/nodejs/bootstrap/pull/65/commits/5a6138e48b7177f2252d5519a786421ce02def45